xarray/core/variable.py

```diff
@@ -192,8 +192,6 @@ def _maybe_wrap_data(data):
     if isinstance(data, pd.Index):
         return PandasIndexingAdapter(data)
     if isinstance(data, pd.api.extensions.ExtensionArray):
-        if isinstance(data.dtype, pd.Int64Dtype | pd.Float64Dtype | pd.StringDtype):
```

Comment:
This casting was intentional. We'd like these dtypes to be converted to numpy arrays.

Reply:
Ok I had lost track of who did this or not, I thought I checked the blame but must not have.  Could explain a bit why so I can add a comment?

Reply:
I think its mostly backwards compatibility, and we aren't sure how to handle the possibility of both numpy and pandas handling very similar dtypes.